infrastructure/variables.tf

```diff
@@ -56,7 +56,7 @@ variable "s2s_name" {
 variable "supported_jurisdictions" {
   type = "list"
   description = "Jurisdictions to be supported by Bulk Scan in the given environment. Bulk Scan will only be able to map these ones to IDAM user credentials"
-  default = ["SSCS", "BULKSCAN", "PROBATE", "DIVORCE", "FINREM"]
+  default = ["SSCS", "BULKSCAN", "PROBATE", "DIVORCE", "FINREM","CMC"]
```

Comment:
If you add it here it will enable on prod

Reply:
oh :(, thank you ,will remove it now.